src/lib/blocks.js

```diff
@@ -356,6 +356,8 @@ export default function (vm, useCatBlocks) {
         };
 
         const json = jsonForSensingMenus(menuFn);
+        json["extensions"].push("output_string");
+        console.log(json);
```

Comment:
Delete `console.log`

Reply:
Done, also removed the extension bit which was unintended. Thanks for the catch!